### ISSUE TITLE
Paket Push doesn't offer a way to specify an alternate symbol server

### ISSUE THREAD

**Issue by nathanaeljones** [fsprojects/Paket#801]
[low priority]

Given that a different server is used when pushing symbol vs. regular nuget packages, we need to be able to override that URL as well as the default. We don't want to push nightly symbols to symbolsource.org when the main packages are only going to MyGet.

**Comment by forki** [comment]
http://fsprojects.github.io/Paket/paket-push.html shows there is a enpoint parameter

**Comment by nathanaeljones** [comment]
Right, but we need two endpoints. One to use for .nupkg files, another for .symbols.nupkg files. At least, if we're passing Paket.Push a directory.

**Comment by forki** [comment]
I see. what defaults should it take?

/cc @mavnn

**Comment by forki** [comment]
@nathanaeljones are you interested in contributing this feature?

**Comment by nathanaeljones** [comment]
Looks like the default should be `http://nuget.gw.symbolsource.org/Public/NuGet` per https://github.com/NuGet/NuGet3/blob/c9aa766effd1b7bb9ea0e50af2822a4a0a8671f9/src/NuGet.Configuration/PackageSource/NuGetConstants.cs#L12 and https://docs.nuget.org/create/creating-and-publishing-a-symbol-package

Given the [naming convention MyGet uses](http://docs.myget.org/docs/reference/symbolsource), it would actually be possible to derive the correct symbol server from the original feed. 
I.e, http://www.myget.org/F/somefeed/api/v1 -> http://nuget.gw.SymbolSource.org/MyGet/somefeed, but this is probably out of scope. 

----

I would be interested in contributing this feature eventually, but I'd like to avoid context switching until ImageResizer 4.0 reaches nuget.org, as that feeds the family. If around that time we have an actionable plan for #714, all the better. If I can make Paket work for my primary use cases, I will be able to invest a lot more time.
